scm-ui/ui-components/src/MarkdownLinkRenderer.tsx

```diff
@@ -75,6 +79,9 @@ const isSubDirectoryOf = (basePath: string, currentPath: string) => {
 };
 
 export const createLocalLink = (basePath: string, currentPath: string, link: string) => {
+  if (isInternalScmRepoLink(link)) {
+    return link;
+  }
```

Comment:
Is an repo link not the same as an absolute link?

Reply:
Actually no, because i don't want to join my path with the basepath. For this special case i just need the relative path, so i can link between different repositories.